api/proto/disperser/disperser.proto

```diff
@@ -11,6 +11,8 @@ service Disperser {
 	// processing status of the blob.
 	rpc DisperseBlob(DisperseBlobRequest) returns (DisperseBlobReply) {}
 
+	rpc DisperseBlobAuthenticated(stream AuthenticatedRequest) returns (stream AuthenticatedReply);
```

Comment:
How does this compare to having a separate interface (like the `GetBlobStatus`) for authentication (and adding auth data to the `DisperserBlobRequest` in the `DisperseBlob` interface)?

Reply:
Interesting idea. In the final flow, we'll likely be having the requester sign the kzg commitment (for payment verification purposes, see doc: https://docs.google.com/document/d/1w2H38txbe21ex8bwxNMAcjRq4K5WHq0NCv6jYy8yo54/edit#heading=h.2ty1ce44eg7b), so I think it is probably necessary to couple them together. Wdyt?

Reply:
Does this coupling need to actually tie the dispersal request and authentication request together, i.e. can they be two interfaces?

It will also be great to document the client-server communication protocol here.

Reply:
Having them coupled allows the following flow: 
- Client sends request
- Disperser calculates kzg commitment and sends back to client with random point opening
- Client verifies kzg commitment (by opening random point) and then signs it 

If we want to have decoupled interfaces, it seems like it would have to be a different authentication model (not signing the kzg commitment.) BUt I think the design document makes a good case for why we will need this is the long term.

Reply:
I'll add some comments to the interface.

Reply:
ok, sg